mindsdb/integrations/handlers/symbl_handler/symbl_tables.py

```diff
@@ -0,0 +1,648 @@
+
+        df = select_statement_executor.execute_query()
+
+        return df
+
+    def parse_response(self, res):
+        return json.loads(json.dumps(res.to_dict(), default=str))
+
+    def get_columns(self) -> List[str]:
+        """Gets all columns to be returned in pandas DataFrame responses
+
+        Returns
+        -------
+        List[str]
+            List of columns
+        """
+
+        return [
+            "id",
+            "text",
+            "type",
+            "score",
+            "message_ids",
+            "entities",
+            "phrases",
+            "definitive",
+            "due_by",
+            "_from.id",
+            "_from.name",
+            "_from.user_id",
+            "assignee.id",
+            "assignee.name",
+            "assignee.email"
+        ]
+
+
+class GetAnalyticsTable(APITable):
+    """The Get Analytics Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Get the analytics for the given conversation id" API
+
+        Parameters
+        ----------
+        query : ast.Select
+           Given SQL SELECT query
+
+        Returns
+        -------
+        pd.DataFrame
+            metrics
+
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'get_analytics',
+            self.get_columns()
+        )
+
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        search_params = {}
+        subset_where_conditions = []
+        for op, arg1, arg2 in where_conditions:
+            if arg1 == 'conversation_id':
+                if op == '=':
+                    search_params["conversation_id"] = arg2
+                else:
+                    raise NotImplementedError("Only '=' operator is supported for conversation_id column.")
+
+            elif arg1 in self.get_columns():
+                subset_where_conditions.append([op, arg1, arg2])
+
+        filter_flag = ("conversation_id" in search_params)
+
+        if not filter_flag:
+            raise NotImplementedError("conversation_id column has to be present in where clause.")
+
+        df = pd.DataFrame(columns=self.get_columns())
+
+        resp = symbl.Conversations.get_analytics(conversation_id=search_params.get("conversation_id"), credentials=self.handler.credentials)
+
+        resp = self.parse_response(resp)
+
+        df = pd.json_normalize(resp["metrics"])
+
+        select_statement_executor = SELECTQueryExecutor(
+            df,
+            selected_columns,
+            subset_where_conditions,
+            order_by_conditions,
+            result_limit
+        )
+
+        df = select_statement_executor.execute_query()
+
+        return df
+
+    def parse_response(self, res):
+        return json.loads(json.dumps(res.to_dict(), default=str))
```

Comment:
what the reason to dump/parse to json?

Reply:
The response from the API is a deeply nested json where timestamp is represented as datetime object. Hence, dump/parse to json.

Reply:
ok, we can left it as is then

Reply:
ok